packages/teleterm/package.json

```diff
@@ -17,7 +17,8 @@
     "build-main": "webpack build --config webpack.main.config.js --progress --mode=production",
     "build-renderer": "webpack build --config webpack.renderer.prod.config.js --progress",
     "build-natives": "electron-builder install-app-deps",
-    "package": "electron-builder build --publish never -c.extraMetadata.name=teleconnect"
+    "package": "electron-builder build --publish never -c.extraMetadata.name=teleconnect",
+    "generate-grpc-shared": "npx -y --target_arch=x64 --package=grpc_tools_node_protoc_ts --package=grpc-tools -- grpc_tools_node_protoc -I=src/sharedProcess/api/proto --ts_out=service=grpc-node,mode=grpc-js:src/sharedProcess/api/protogen --grpc_out=grpc_js:src/sharedProcess/api/protogen --js_out=import_style=commonjs,binary:src/sharedProcess/api/protogen src/sharedProcess/api/proto/*.proto"
```

Comment:
Is the name `generate-grpc-shared` used to make it more clear that we don't mean the gRPC files behind daemon <=> Connect communication?

I think we should add information about this command to the readme next to that diagram.

Reply:
Done